src/main/flight/mixer_init.c

```diff
@@ -370,7 +370,9 @@ void mixerInitProfile(void)
 void mixerResetRpmLimiter(void)
 {
     const float maxExpectedRpm = MAX(1.0f, motorConfig()->kv * getBatteryVoltage() * 0.01f);
-    mixerRuntime.rpmLimiterThrottleScale = constrainf(mixerRuntime.rpmLimiterRpmLimit / maxExpectedRpm, 0.0f, 1.0f);
+// Empirical testing found this linear relationship between expected RPM and measured efficiency, independent from prop size
+const float rpmDerating = -5.44e-6 * maxExpectedRpm + 0.944;
```

Comment:
```suggestion
    // Empirical testing found this linear relationship between expected RPM and measured efficiency, independent from prop size
    const float rpmDerating = -5.44e-6 * maxExpectedRpm + 0.944;
```

Reply:
Just noticed that the numerical constants are explicitly `double`. Just add an `f` to the end for explicit float values 👍
```suggestion
    // Empirical testing found this linear relationship between expected RPM and measured efficiency, independent from prop size
    const float rpmDerating = -5.44e-6f * maxExpectedRpm + 0.944f;
```